Assets/HoloToolkit-Examples/SpatialUnderstanding/SpatialUnderstanding-FeatureOverview/Scripts/SpatialUnderstandingBasicCursor.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using HoloToolkit.Unity.InputModule;
+using UnityEngine;
+
+/// <summary>
+/// 1. Decides when to show the cursor.
+/// 2. Positions the cursor at the gazed location.
+/// 3. Rotates the cursor to match hologram normals.
+/// </summary>
+public class SpatialUnderstandingBasicCursor : MonoBehaviour
```

Comment:
I can add it if you want, but it was never in there in the first place. Your call.
In any case, it would be HoloToolkit.Unity.Examples, or something like that?